src/services/apiService.js

```diff
@@ -217,9 +217,11 @@ angular.module('openshiftCommon')
       if (!_.get(APIS_CFG, ["groups", resource.group, "versions", resource.version, "resources", primaryResource])) {
         return undefined;
       }
+      var hostPrefixObj = _.get(APIS_CFG, ["groups", resource.group, 'hostPrefix']) || APIS_CFG;
```

Comment:
Nit: Since `_.get` takes a default, I'd use that.

```js
var hostPrefixObj = _.get(APIS_CFG, ["groups", resource.group, 'hostPrefix'], APIS_CFG);
```